### PR TITLE
Hello mIcroverse

### PR DESCRIPTION
**Pull request by Hassaanjbaig-code**
[] This is my project which hello microvers

[] Create a repo 
[] Create a gitflow
[] Added linters
[]Add HTML and CSS

### CODE REVIEW

**Comment by Hassaanjbaig-code** [comment]
All the file are commit

File: .github/workflows/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/node_modules
+/test.md
```
Comment (pinkmoon25) [inline .github/workflows/.gitignore:2]: your .github/workflows folder should contain only linters.yml file inside of it, please remove .gitignore outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put .gitignore inside hello-microverse.


- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md

File: .github/workflows/.hintrc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```
Comment (pinkmoon25) [inline .github/workflows/.hintrc:18]: your .github/workflows folder should contain only linters.yml file inside of it, please remove .hintrc outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put .hintrc inside hello-microverse.


- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md

File: .github/workflows/.stylelintrc.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            "variants",
+            "responsive",
+            "screen"
+          ]
+        }
+      ]
+    },
+    "csstree/validator": true,
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  }
```
Comment (pinkmoon25) [inline .github/workflows/.stylelintrc.json:21]: your .github/workflows folder should contain only linters.yml file inside of it, please remove .stylelintrc.json outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put .stylelintrc.json inside hello-microverse.

file structure :point_right:  

- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md

File: .github/workflows/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (pinkmoon25) [inline .github/workflows/README.md:63]: your .github/workflows folder should contain only linters.yml file inside of it, please remove ReadMe.md outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put ReadMe.md inside hello-microverse.


- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md

File: .github/workflows/LICENSE [hunk 5]
```diff
@@ -0,0 +1,7 @@
+## Copyright 2021, [Hassaanjbaig-code]
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
+
+THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```
Comment (pinkmoon25) [inline .github/workflows/LICENSE:7]: your .github/workflows folder should contain only linters.yml file inside of it, please remove Licence outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put License inside hello-microverse.


- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md

File: .github/workflows/index.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello MIcroverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="header">Hello Microverse</h1>
+</body>
+</html>
```
Comment (pinkmoon25) [inline .github/workflows/index.html:13]: your .github/workflows folder should contain only linters.yml file inside of it, please remove index.html outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put index.html inside hello-microverse.


- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md

File: .github/workflows/package.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "homepage": "https://github.com/Hassaanjbaig-code/Hello--MIcroverse-#readme",
+  "devDependencies": {
+    "hint": "^7.1.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```
Comment (pinkmoon25) [inline .github/workflows/package.json:27]: your .github/workflows folder should contain only linters.yml file inside of it, please remove package.json and package-lock.json outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put  package.json and package-lock.json inside hello-microverse.


- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md
  - package.json
  - package-lock.json

File: .github/workflows/style.css [hunk 8]
```diff
@@ -0,0 +1,3 @@
+.header {
+  color: red;
+}
```
Comment (pinkmoon25) [inline .github/workflows/style.css:3]: your .github/workflows folder should contain only linters.yml file inside of it, please remove style.css outside of this directory and put it on your root folder
example: if your root folder name is hello-microverse put  style.css inside hello-microverse.


- root_folder 
  -   .github/workflows
         -  linters.yml
  -   .stylelint.json
  -   .hintrc
  -   .gitignore
  -   index.html
  -   style.css
  - test.md
  - package.json
  - package-lock.json

**Review by pinkmoon25 — COMMENTED** [review]
Hi @Hassaanjbaig-code , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 


## Highlights

- Followed github flow :heavy_check_mark: 
- Included index.html :heavy_check_mark: 
- included style.css :heavy_check_mark: 
- Added a professional readme file :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

- [ ]  Please add a test.md file with the text "This file should be ignored by git".
- [ ] Please do not close or merge a  pull request before a code review 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**